js_modules/dagit/packages/ui/src/components/calculateMiddleTruncation.tsx

```diff
@@ -1,7 +1,7 @@
 // We've observed that Firefox's DOM APIs and Canvas APIs do not return
 // identical values given the same rendered text, in particular when the DOM
 // element is inside a flexbox. They're floating point numbers off by ~0.05.
-const FIREFOX_WIDTH_VARIANCE = 0.1;
+const FIREFOX_WIDTH_VARIANCE = 0.2;
```

Comment:
you high confidence on this?

Reply:
Hmm, I think this behavior is undesirable but it's not a recent regression -- let me undo this change here and I'll submit a separate PR for it. This is more "would be nice to fix for next release", and I'm actually not confident that 0.2 is high enough to fully eliminate the problem.